pkg/apigee/apigee.go

```diff
@@ -1,45 +1,78 @@
 package apigee
 
 import (
-	"os"
-	"fmt"
 	"errors"
+	"fmt"
+	"os"
 	"sync"
 
-	"net/http"
 	"encoding/json"
+	"net/http"
 )
 
 const (
-	// Default Apigee's api endpoint host
+	// DefaultApigeeHost is Apigees defai;t api endpoint host
 	DefaultApigeeHost = "https://api.enterprise.apigee.com/"
 
-	// Env Var to set overide default apigee api host
+	// EnvVarApigeeHost is the Env Var to set overide default apigee api host
 	EnvVarApigeeHost = "AUTH_API_HOST"
 )
 
+// Client is the client struct for interacting with external Apigee APIs
 type Client struct {
 	// Authorization token used in the Authorization header on each request
-	Token   string
-	
+	Token string
+
 	// Apigee api host
-	ApigeeApiHost  string
-	
-	// Shared httpClient for efficiency
-	HttpClient *http.Client
+	ApigeeAPIHost string
+
+	// Shared HTTPClient for efficiency
+	HTTPClient *http.Client
 }
 
+// GetKVM returns a map that corresponds to the key:value of a named LVM
+func (c *Client) GetKVM(org, env, kvmName string) (map[string]string, error) {
```

Comment:
I see you're pulling down the whole map each time instead of gathering only the value that is necessary.  Would it make sense to use the _[get map entry API](http://docs.apigee.com/management/apis/get/organizations/%7Borg_name%7D/keyvaluemaps/%7Bmap_name%7D/entries/%7Bentry_name%7D)_ and only get the values we need?

_(I've not read the rest of the PR yet so this could end up being moot.)_